### DISCUSSION TITLE
Documentación de Metabase para su fácil traspaso a clientes

### DISCUSSION THREAD

**Discussion by AlexBaader** [Cotalker/documentation#987] [3. Ideas & Feature Requests]
Hola a todos!

últimamente ha surgido el requisito de algunos clientes de poder administrar por su cuenta sus dashboards. Para ello es necesario realizar una breve capacitación en Metabase, centrada principalmente en la estructura de datos de cotalker (qué tablas consultar para obtener que respuestas). Por lo que sería de gran utilidad contar con una documentación genérica. 

A modo de ejemplo dejo el siguiente documento:

https://docs.google.com/document/d/1wfsT26o-hY3OVK5xlnN9CbzisYb6SLDVXpJ-5QqPWOM/edit#heading=h.kqx3av8thbn1

**Comment by acedward** [comment]
Esto ya existe. Hay una herramienta interna para generar el documento.

**Reply by jdurandpaz** [reply]
@acedward pero la parte de la explicación? la cubre el doc, yo creo que sería bueno tener enla documentación como una explicación muy global, tablas de formularios, tablas de propiedades , tablas de tareas, tasklogs, etc.

**Reply by acedward** [reply]
https://doc.cotalker.com/docs/documentation/sql_bi/model
